modules/datadog-slo/asm.tf

```diff
@@ -0,0 +1,19 @@
+data "aws_secretsmanager_secret" "datadog_api_key" {
+  count = var.secrets_store_type == "ASM" ? 1 : 0
```

Comment:
Please include `local.enabled` logic here so this can be disabled

Reply:
@Benbentwo please review @nitrocode  comments